optimade/server/routers/info.py

```diff
@@ -34,6 +38,13 @@
 def get_info(request: Request):
     from optimade.models import BaseInfoResource, BaseInfoAttributes
 
+    parse_result = urllib.parse.urlparse(str(request.url))
+    base_url = (
+        CONFIG.base_url
+        if CONFIG.base_url
+        else f"{parse_result.scheme}://{parse_result.netloc}"
```

Comment:
Oh, so you get the base URL from the request?

I wonder - would that solve the problem of having to specify the base URL?

Reply:
No.
The `else` value is what used to be up to now, and is giving false information if run in a docker (it will provide you with something like `http://localhost:3585` or whatever the port is).